### DISCUSSION TITLE
这个项目是长期维护的吗？敢用于生产环境吗？

### DISCUSSION THREAD

**Discussion by hooliy** [mallfoundry/taroify#835] [Q&A]
与nutui-react-taro比较哪个更好？

**Comment by CodeMak1r-zy** [comment]
nutui-react-taro更多的是京东风格，taroify风格比较中立。
长期维护中，有问题可以加群交流，或者提issue。 生产环境当然是可以用的

**Reply by hooliy** [reply]
好的，谢谢！nutui-react-taro 用了一段时间，感觉还是有些bug，准备把项目切换到这个上面试试。
